.github/workflows/5_rbac-defaults-version-guard.yml

```diff
@@ -0,0 +1,82 @@
+name: 5.X - RBAC (Framework) - Defaults version guard
```

Comment:
IMHO RBAC is more likely to be related to the API than the framework

Reply:
Fixed at https://github.com/wazuh/wazuh/pull/33003/commits/07883718f22f77c3d729dfb905ab151087fb6a95